recipes/capella-console-client/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "capella-console-client" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/capella-console-client-{{ version }}.tar.gz
+  sha256: f587b89c7a2fb37b85b7b9a0424acea500d7f82f5ff05f498cfabec55fc46c5a
+
+build:
+  entry_points:
+    - capella-console-wizard = capella_console_client.cli.wizard:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry
```

```suggestion
    - poetry-core >=1.0.0
```
Ref: https://github.com/capellaspace/console-client/blob/bad631f207043231630c9b8c0893a1b4382d4061/pyproject.toml#L64